Documentation/ApiOverview/ContentElements/Index.rst

```diff
@@ -110,21 +110,86 @@ your projects.
 possible without writing PHP code and can therefore also be done by
 TYPO3 integrators.
 
+..  _plugins:
 
+Plugins in TYPO3
+----------------
+
+A **plugin** in TYPO3 is a more complex module, typically providing dynamic
```

Comment:
A "module" is a special name for own implementations in TYPO3 backend. I would try to prevent using that word in FE context. What about implementation or integration?

```suggestion
A **plugin** in TYPO3 is a more complex implementation, typically providing dynamic
```